src/components/shared/Layer.js

```diff
@@ -0,0 +1,13 @@
+import { Layer as GrommetLayer } from 'grommet'
+import { useResponsive } from 'hooks/useResponsive'
+
+export const Layer = ({ position = '', show = false, children }) => {
+  const { viewport } = useResponsive()
+  if (viewport !== 'small') return children
+  return (
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    <>{show && <GrommetLayer position={position}>{children}</GrommetLayer>}</>
```

Comment:
```suggestion
    if (show) return (<GrommetLayer position={position}>{children}</GrommetLayer>)
    return </>
```

If you write it this way you dont have to wrap a single component in a fragment.

Reply:
This might need to be linted so I wouldn't commit this here

Reply:
Revised, thank you!

**NOTE:**
- Returning the `null` instead of `</>` to avoid having the same eslint error for `jsx-no-useless-fragment`
- The same update was also applied to the latest [`Layer`](https://github.com/AlexsLemonade/refinebio-web/blob/4fe799a11692b236acb47433bbbccd4ef52c784e/src/components/shared/Layer.js) component in PR [37 - Add Responsive to Search Result Page](https://github.com/AlexsLemonade/refinebio-web/pull/38)